### PR TITLE
interfaces/browser-support: allow editing of Jupyter notebooks in browsers

### PR DESCRIPTION
**Pull request by nteodosio**
Read-write access is needed to edit Jupyter notebooks. 

https://bugs.launchpad.net/ubuntu/+source/jupyter-notebook/+bug/1959417

### CODE REVIEW

File: interfaces/builtin/browser_support.go [hunk 1]
```diff
@@ -72,6 +72,9 @@ owner /{dev,run}/shm/.io.nwjs.* mrw,
 /run/user/[0-9]*/snap.@{SNAP_INSTANCE_NAME}/{,.}com.google.Chrome.*/SS r,
 /run/user/[0-9]*/snap.@{SNAP_INSTANCE_NAME}/{,.}com.microsoft.Edge.*/SS r,
 
+# Allow access to Jupyter notebooks (LP: #1959417)
+owner @{HOME}/.local/share/jupyter/** rw,
```
Comment (mardy) [inline interfaces/builtin/browser_support.go:76]: I would suggest to add a comment here saying that this access is temporary, and only granted until [bug 1959417](https://bugs.launchpad.net/ubuntu/+source/jupyter-notebook/+bug/1959417) is fixed upstream.

At the same time we should start working with upstream to either:
1. Change the path where notebooks are generated, or
2. Change the text displayed to not invite the user to open that path with the browser

Comment (mardy) [reply interfaces/builtin/browser_support.go:76]: I'm trying to get in touch with the jupiter-notebook developers here: https://github.com/jupyter/notebook/discussions/6436

**Review by alexmurray — APPROVED** [review]
LGTM but I wonder if this is a slippery slope where we then need to start adding other similar accesses for other applications' data?

**Comment by seb128** [comment]
> LGTM but I wonder if this is a slippery slope where we then need to start adding other similar accesses for other applications' data?

I don't think we should start adding things to that list but rather work with upstream to get issues resolved. In this case it's a visible regression in a popular software so it makes sense to mitigate the issue we created by removing the deb, at least until a proper solution is worked out upstream

**Review by mardy — APPROVED** [review]
Thanks, LGTM!

**Review by bboozzoo — APPROVED** [review]
Thank you

**Comment by bboozzoo** [comment]
I cannot push to this PR, so I've opened #11847 with an update to the relevant spread test.

**Pull request merged**